xml/vk.xml

```diff
@@ -11222,6 +11222,12 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="0"                                             name="VK_EXT_EXTENSION_285_SPEC_VERSION"/>
                 <enum value="&quot;VK_EXT_extension_285&quot;"              name="VK_EXT_extension_285"/>
             </require>
+        </extension>
+		<extension name="VK_EXT_extension_286" number="286" type="instance" author="MESA" contact="Drew DeVault <sir@cmpwn.com>" supported="disabled">
```

Comment:
I think your text editor had a minor issue here and the indentation got messed up.  We use spaces, not tabs.

Reply:
Looks good to me now.